sdk/core/corehttp/corehttp/exceptions.py

```diff
@@ -200,6 +202,8 @@ def __init__(
             self.reason = response.reason
             self.status_code = response.status_code
 
+        self.model: Optional[Any] = kwargs.pop("model", None)
```

Comment:
I don't have better typing advise for the model, so `Any` is fine to me for now.